app/atomic_svc.py

```diff
@@ -292,6 +292,9 @@ async def _save_ability(self, entries, test):
                 command, cleanup, payloads = await self._prepare_executor(test, platform, executor)
                 data['platforms'][platform] = dict()
                 data['platforms'][platform][executor] = dict(command=command, payloads=payloads, cleanup=cleanup)
+                if executor == 'psh':
+                    data['platforms'][platform][executor]['parsers'] = {'plugins.atomic.app.parsers.powershell':
+                                                                        [{'source': 'validate_me'}]}
```

Comment:
This is fine as a source name, but is there a better one? Like `error` or `powershell.error` or `atomic_powershell`?

Reply:
I'll change it to atomic_powershell (I'm not sure you can use `.` like that, and error is too generic).